### ISSUE TITLE
pyimagej -> xarray ->zarr

### ISSUE THREAD

**Issue by Sh4zKh4n** [imagej/pyimagej#124]
I have got pyimagej to open a .lif file then converting to an xarray dataarray, though in reality most of the meta data is a bit incomprehensible. The issue I have is that this is coming out quite confusing. I know I have a number of different stacks in the file but xarray seems to be reading them as something that isnt the same as a group of stacks. Since I would prefer to save these as a zarr file format. I have tried to save through the dataset command. So converted to a dataset but then got an error on attributes Does anyone have any thoughts? I would have thought that since many file formats contain different stacks like T or just different sequences, that when Loci opens these and pyimagej would convert to a dataset instead of a dataarray. Not really sure what to do at this point? I just want to extract and don't mind noting the xyz dims meta data, the only section I really care about. Any thoughts?

**Comment by joshmoore** [comment]
Hi @Sh4zKh4n. Not knowing what exactly your use case is, this may not be ideal for you, but I'd suggest using https://github.com/glencoesoftware/bioformats2raw outside of pyimagej to get an OME-Zarr that you could then make use of.

**Comment by ctrueden** [comment]
@Sh4zKh4n Would it be possible for you to post a sample file and Python script, illustrating what you are trying to do, and what goes wrong?

**Comment by Sh4zKh4n** [comment]
@ctrueden the file size isn't small, it's a series of 3 stacks that are over 10GB's all together as .lif files. 

I have had to fall back onto another library called readlif, which has the same issue on metadata. Yet I can at least access and save the each stack as a zarr file. The metadata, is unfortunately lost. It's a very strange 

The code I used was the the standard method used according to the pyimage docs/examples.

I think what's gone wrong is that the 3 different files are not the same. One is a 'lamda' stack to scan through excitation wavelengths (this particular dataset is a multiphoton microscope) and the next two are standard volumes. 

In any case, I have a solution for now.
